### PR TITLE
*: add region read/write bytes info and approximate size/keys in show table regions result.

### PR DESCRIPTION
**Pull request by crazycs520**
### What problem does this PR solve? 
*  Add region read/write bytes info and approximate size/keys in show table regions result.

before

```sql
test> show table t_wide regions;
+-----------+----------------------------+----------------------------+-----------+-----------------+---------------+------------+
| REGION_ID | START_KEY                  | END_Key                    | LEADER_ID | LEADER_STORE_ID | PEERS         | SCATTERING |
+-----------+----------------------------+----------------------------+-----------+-----------------+---------------+------------+
| 208       | t_49_                      | t_49_r_1152921504606846976 | 211       | 11              | 209, 210, 211 | 0          |
| 212       | t_49_r_1152921504606846976 | t_49_r_2305843009213693952 | 216       | 3               | 215, 217, 216 | 0          |
| 218       | t_49_r_2305843009213693952 | t_49_r_3458764513820540928 | 222       | 1               | 219, 220, 222 | 0          |
| 223       | t_49_r_3458764513820540928 | t_49_r_4611686018427387904 | 227       | 3               | 225, 226, 227 | 0          |
| 228       | t_49_r_4611686018427387904 | t_49_r_5764607523034234880 | 233       | 3               | 233, 232, 283 | 0          |
| 234       | t_49_r_5764607523034234880 | t_49_r_6917529027641081856 | 236       | 10              | 236, 258, 264 | 0          |
| 238       | t_49_r_6917529027641081856 | t_49_r_8070450532247928832 | 267       | 10              | 243, 242, 267 | 0          |
| 5         | t_49_r_8070450532247928832 |                            | 255       | 1               | 96, 255, 256  | 0          |
+-----------+----------------------------+----------------------------+-----------+-----------------+---------------+------------+
```

In This PR

```sql
+-----------+----------------------------+----------------------------+-----------+-----------------+---------------+------------+---------------+------------+------------------+------------------+
| REGION_ID | START_KEY                  | END_KEY                    | LEADER_ID | LEADER_STORE_ID | PEERS         | SCATTERING | WRITTEN_BYTES | READ_BYTES | APPROXIMATE_SIZE(MB) | APPROXIMATE_KEYS |
+-----------+----------------------------+----------------------------+-----------+-----------------+---------------+------------+---------------+------------+------------------+------------------+
| 208       | t_49_                      | t_49_r_1152921504606846976 | 211       | 11              | 209, 210, 211 | 0          | 21955670      | 34808000   | 129              | 109974           |
| 212       | t_49_r_1152921504606846976 | t_49_r_2305843009213693952 | 216       | 3               | 215, 217, 216 | 0          | 18482953      | 40228600   | 59               | 220528           |
| 218       | t_49_r_2305843009213693952 | t_49_r_3458764513820540928 | 222       | 1               | 219, 220, 222 | 0          | 21412156      | 40476040   | 76               | 74187            |
| 223       | t_49_r_3458764513820540928 | t_49_r_4611686018427387904 | 227       | 3               | 225, 226, 227 | 0          | 20311151      | 35104000   | 37               | 30999            |
| 228       | t_49_r_4611686018427387904 | t_49_r_5764607523034234880 | 233       | 3               | 233, 232, 283 | 0          | 0             | 0          | 37               | 78094            |
| 234       | t_49_r_5764607523034234880 | t_49_r_6917529027641081856 | 236       | 10              | 236, 258, 264 | 0          | 0             | 0          | 33               | 36885            |
| 238       | t_49_r_6917529027641081856 | t_49_r_8070450532247928832 | 267       | 10              | 243, 242, 267 | 0          | 0             | 0          | 40               | 51768            |
| 5         | t_49_r_8070450532247928832 |                            | 255       | 1               | 96, 255, 256  | 0          | 0             | 0          | 33               | 33000            |
+-----------+----------------------------+----------------------------+-----------+-----------------+---------------+------------+---------------+------------+------------------+------------------+
```

**Add 4 new field**
* `WRITTEN_BYTES`: the write bytes in the heartbeat cycle.  unit is bytes.
* `READ_BYTES `:   the read bytes in the heartbeat cycle.  unit is bytes.
* `APPROXIMATE_SIZE(MB) `:  approximate region size.  unit is MB.
* `APPROXIMATE_KEYS`: approximate region keys.

### What is changed and how it works?
Use  PD HTTP API `/pd/api/v1//region/id/{id}`  to get region info  one by  one. 

### problem

**The region info got from  PD may not be accurate.**

### Check List 

Tests 

 - Unit test

Code changes

 - Has exported function/method change

Side effects

Related changes

 - Need to cherry-pick to the release branch

Release Note

- Add more region flow information in show table region result.

### CODE REVIEW

File: util/pdapi/const.go [hunk 1]
```diff
@@ -15,8 +15,9 @@ package pdapi
 
 // The following constants are the APIs of PD server.
 const (
-	HotRead  = "/pd/api/v1/hotspot/regions/read"
-	HotWrite = "/pd/api/v1/hotspot/regions/write"
-	Regions  = "/pd/api/v1/regions"
-	Stores   = "/pd/api/v1/stores"
+	HotRead    = "/pd/api/v1/hotspot/regions/read"
```
[the remark this answers is not in the document]
Comment (crazycs520) [reply util/pdapi/const.go]: yep. I will do it.

**Comment by winkyao** [comment]
Could you change APPROXIMATE_SIZE to `APPROXIMATE_SIZE (MB)`？

**Pull request merged**